src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -137,37 +137,29 @@ private static String generateCvrSnapshotId(String cvrId, Integer round) {
 
   // generates an internal ContestSelectionId based on a candidate code
   private static String getCdfContestSelectionIdForCandidateCode(String code) {
-    String id = cdfCandidateCodeToContestSelectionId.get(code);
-    if (id == null) {
-      id = String.format("cs-%s", sanitizeStringForOutput(code).toLowerCase());
-      cdfCandidateCodeToContestSelectionId.put(code, id);
-    }
-    return id;
+    return cdfCandidateCodeToContestSelectionId.computeIfAbsent(code,
+        c -> String.format("cs-%s", sanitizeStringForOutput(c).toLowerCase()));
   }
 
   // generates an internal CandidateId based on a candidate code
   private static String getCdfCandidateIdForCandidateCode(String code) {
-    String id = cdfCandidateCodeToCandidateId.get(code);
-    if (id == null) {
-      id = String.format("c-%s", sanitizeStringForOutput(code).toLowerCase());
-      cdfCandidateCodeToCandidateId.put(code, id);
-    }
-    return id;
+    return cdfCandidateCodeToCandidateId.computeIfAbsent(code,
+        c -> String.format("c-%s", sanitizeStringForOutput(c).toLowerCase()));
   }
 
   // Instead of a map from rank to list of candidates, we need a sorted list of candidates
   // with the ranks they were given. (Ordinarily a candidate will have only a single rank, but they
   // could have multiple ranks if the ballot duplicates the candidate, i.e. assigns them multiple
-  // ranks.
+  // ranks).
```

Comment:
super nit: close parens should be after the period

Reply:
Done.